vcr/deploy/deploy.go

```diff
@@ -257,6 +256,11 @@ func compressDir(source string) (int, []byte, error) {
 		if info.IsDir() {
 			return nil
 		}
+
+		if !hasReadAndWritePermission(path) {
+			fmt.Fprintf(io.ErrOut, "%s Skipping file %q: no read and write permission\n", c.WarningIcon(), path)
```

Comment:
i think you should only print this out when files do not begin with .jfs, otherwise people will just see this all the time and it's kind of confusing